command/sub8_missions/nodes/move_command

```diff
@@ -37,8 +47,49 @@ def main(args):
         fprint("Waiting for odom...")
         yield sub.tx_pose()
         fprint("Odom found!", msg_color='green')
-
+        rospack = rospkg.RosPack()
+        config_file = os.path.join(rospack.get_path('sub8_missions'), 'nodes', 'tp.yaml')
+        f = yaml.load(open(config_file, 'r'))
         action_kwargs = {'speed': float(args.speed), 'blind': bool(args.blind)}
+        if args.gteleport:
+            try:
+                state_set_pub = yield nh.advertise('gazebo/set_model_state', ModelState)
+                try:
```

Comment:
Generally you don't use try except for stuff like this. Better to check if it is valid manually

```
split_args = args.gteleport.split()
if len(split_args) == 3
    # run as coordinates
else:
    # check if in yaml
```